<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogbookWidget</class>
 <widget class="QWidget" name="LogbookWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableView" name="tableView">
     <property name="font">
      <font/>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="defaultPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="flightSearchLabel">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Search Flight</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="flightSearchLlineEdit">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter the searchterm you want to filter your flights by.&lt;/p&gt;&lt;p&gt;For dates, make sure to use the format YYYY-MM-DD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="flightSearchInLabel">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Search by</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="flightSearchComboBox">
         <item>
          <property name="text">
           <string>Date of Flight</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Departure Airport</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Destination Airport</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Aircraft Registration</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pilot Name</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select a Flight from the list to show or edit details.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="viewsComboBox"/>
   </item>
  </layout>
  <widget class="QLabel" name="spacerLabel">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>410</y>
     <width>200</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <action name="actionEdit_Flight">
   <property name="text">
    <string>Edit Flight</string>
   </property>
  </action>
  <action name="actionDelete_Flight">
   <property name="text">
    <string>Delete Flight</string>
   </property>
  </action>
  <action name="actionEdit_Sim">
   <property name="text">
    <string>Edit_Sim</string>
   </property>
   <property name="toolTip">
    <string>Edit an existing Simulator Entry</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
